site/src/pages/TemplateSettingsPage/TemplateSchedulePage/TemplateScheduleForm.tsx

```diff
@@ -514,16 +517,26 @@ export const TemplateScheduleForm: FC<TemplateScheduleForm> = ({
                 />
               </Stack>
 
-              <Stack>
+              <Stack spacing={DORMANT_FIELDSET_SPACING}>
                 <FormControlLabel
                   control={
                     <Switch
+                      size="small"
                       name="dormancyAutoDeletion"
                       checked={form.values.dormant_autodeletion_cleanup_enabled}
                       onChange={handleToggleDormantAutoDeletion}
                     />
                   }
-                  label="Enable Dormancy Auto-Deletion"
+                  label={
+                    <StackLabel>
+                      Enable Dormancy Auto-Deletion
+                      <StackLabelHelperText>
+                        When enabled, Coder will permanently delete dormant
+                        workspaces after a period of time. Once a workspace is
+                        deleted it cannot be recovered.
+                      </StackLabelHelperText>
```

Comment:
This is good. Can we do text decoration? If it's a small lift, would like to make the following **bold**:

> Once a workspace is deleted it cannot be recovered.

If it's more than a one-line change we don't have to worry about it.

Reply:
What about making it bold + a warning color? 

<img width="715" alt="Screenshot 2024-04-26 at 12 53 32" src="https://github.com/coder/coder/assets/3165839/c4e58a3c-b79d-414a-a47e-a1124991f5f0">